api/src/tasks/import-task.js

```diff
@@ -173,6 +173,10 @@ async function runTask(nrptiDataSource, auth_payload, params = null, recordTypes
     additionalReadRoles.push(ApplicationRoles.ADMIN_FLNRO)
   }
 
+  if(nrptiDataSource.dataSourceLabel == 'mis-csv' && params == 'Inspection'){
+    additionalReadRoles.push(ApplicationRoles.ADMIN_AGRI)
```

Comment:
Missing the AGRI CMDB source here as well

Reply:
Fixed in a41b100